source/slang/glsl.meta.slang

```diff
@@ -2231,107 +2231,239 @@ __generic<T:__BuiltinFloatingPointType, let N:int>
 [require(glsl_hlsl_spirv, texture_sm_4_1_fragment)]
```

Comment:
In general for all texture functions, I wonder if we should use C++  code gen (`${{{{ }}}}`) for these texture functions since they are repetitive. May be easier to iterate on them?

Reply:
I think that is a good idea.
But the `default` part differs on all cases.
I can make it working but I am afraid that it may make things harder to read.